Source/Analytics/Read/CaseReports/CaseReportEventProcessor.cs

```diff
@@ -159,11 +161,13 @@ public void InsertPerHealthRiskAndRegionForDay(CaseReport report, HealthRisk hea
                     if (reportsPerRegion.TryGetValue(region.Name, out int totalReports))
                     {
                         totalReports += numberOfReports;
+                        reportsPerRegion[region.Name] = totalReports;
```

Comment:
can this be simplified to be one line? I can't see that totalReports is being used later.
reportsPerRegion[region.Name] = totalReports + numberOfReports?